src/NuGet.Config

```diff
@@ -1,22 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
```

Comment:
>configuration [](start = 1, length = 13)

Wait, what?? That's not what I did. I deleted the copy under src. I didn't edit it.

Reply:
Fixed.

---
In reply to: [211774294](https://github.com/Microsoft/sarif-sdk/pull/1042#discussion_r211774294) [](ancestors = 211774294)